2025-01-28 20:09:35,113 - ERROR - Exception while fetching token: HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /jwt-api-token-auth/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FF44883B20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
2025-01-28 20:11:57,125 - ERROR - Exception while fetching token: HTTPConnectionPool(host='127.0.0.1', port=8081): Max retries exceeded with url: /jwt-api-token-auth/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D1D125FEE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
